libraries/engage/filter/components/FilterPageContent/components/ApplyButton/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import { I18n, Button, SurroundPortals } from '@shopgate/engage/components';
+import { PORTAL_FILTER_APPLY_BUTTON } from '@shopgate/engage/filter/constants';
+import { withWidgetSettings } from '@shopgate/engage/core';
+import styles from './style';
+
+/**
+ * The filter apply button component.
+ * @param {Object} props The component props
+ * @param {boolean} props.disabled Whether the button is disabled
+ * @param {Function} props.onClick Click handler for the button
```

Comment:
i didn't add it, since it's not a prop that's explicitly passed to the component, since it's injected by the `withWidgetSettings` HOC. adding it to the JSDoc would cause that it comes visible in the type hints where the component is used.